### ISSUE TITLE
Could I share your trained resnet-152-kinetics-ucf101_split1.pth and resnet-152-kinetics-hmdb51_split1.pth models?

### ISSUE THREAD

**Issue by junmuzi** [kenshohara/3D-ResNets-PyTorch#90]
Hi @kenshohara ,

Thanks for your great work. Sorry for trouble you again. Could I share your trained
resnet-152-kinetics-ucf101_split1.pth and resnet-152-kinetics-hmdb51_split1.pth models?

Or if 152 is so large, resnet-101-kinetics-ucf101_split1.pth and resnet-101-kinetics-hmdb51_split1.pth are ok, too.

Thank you very much. :-)

**Comment by junmuzi** [comment]
@all,

Could I share your downloads, anyone who has downloaded above models before? Thanks very much.

**Comment by kenshohara** [comment]
I can take time next week.
Please wait a moment.

**Comment by junmuzi** [comment]
@kenshohara ,

Thank you very much. I want to finetuning a resnet-101-64f-kinetics-hmdb51_split1.pth model based your trained resnet-101-kinetics.pth, could you give me some suggestions? Thanks.

The follow is my cli:
CUDA_VISIBLE_DEVICES=1 python main.py --video_path /root/data_30/hmdb51 --annotation_path /root/data_30/hmdb51_1.json --result_path results_hmdb51-64f-all-lr0.001/train --dataset hmdb51 --model resnet --model_depth 101 --n_classes 400 --n_finetune_classes 51 --batch_size 10 --n_threads 32 --checkpoint 5 --pretrain_path models/resnet-101-kinetics.pth --resnet_shortcut B --sample_duration 64 --learning_rate 0.001 --weight_decay 0.00001 --n_val_samples 1

Current testing accuracy(on hmdb51 split1, the 50th epoch):
top1: 61.6
top5: 89.4
----------------------------------------------------------

I also do another try, this time I use "--ft_begin_index 4" parameter, but the testing result is worse as follows show.

cli:
CUDA_VISIBLE_DEVICES=0 python main.py --video_path /root/data_30/hmdb51 --annotation_path /root/data_30/hmdb51_1.json --result_path results_hmdb51-64f-lr0.1/train --dataset hmdb51 --model resnet --model_depth 101 --n_classes 400 --n_finetune_classes 51 --batch_size 10 --n_threads 32 --checkpoint 5 --pretrain_path models/resnet-101-kinetics.pth --resnet_shortcut B --sample_duration 64 --weight_decay 0.00001 --ft_begin_index 4 --n_val_samples 1

Current testing accuracy(on hmdb51 split1, the 50th epoch):
top1: 61.0
top5: 88.1

**Comment by junmuzi** [comment]
@kenshohara ,

Thank you very much. I have downloaded the models. :-)
